stats/DataAggregation.md

```diff
@@ -0,0 +1,69 @@
+# Data Aggregation API Overview
+The stats library MUST allow users to define measurements how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library MUST provide support for multiple types of Aggregations:
+* `Count`: indicates that data collected and aggregated with this `Aggregation` type will be turned
+into a count value.
+* `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
+* `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
+mean value.
+* `Distribution`: indicates that the desired `Aggregation` is a histogram distribution. A
+distribution `Aggregation` may contain a histogram of the values in the population. User MUST
+define the bucket boundaries for that histogram.
+
+## View
+A View describes how individual measurements are both broken down (by a set of tag keys, which
+form the basis for a Metric labels.) and aggregated (e.g. into Distributions). A single Measure
```

Comment:
In the metric systems usually the common word is label. Here I tried to show people the correlation between OpenCensus tags and Metric label.

Reply:
I think it would help to make it clearer that this is a comparison with other systems, for example, you could say that tags are similar to labels in some other systems.  Otherwise, it sounds like the document is introducing more OpenCensus terminology.  The same applies to "metric" in other parts of the document.

Reply:
The first sentence in Export.md also uses metrics and labels without mentioning that they are from other projects.